src/tests/apitests.ts

```diff
@@ -189,7 +189,7 @@ describe("Laputin API", function() {
         rimraf.sync(archivePath);
         fs.mkdirSync(archivePath);
 
-        var laputin = compose(archivePath, new LaputinConfiguration(1234, "accurate"));
+        var laputin = compose(archivePath, new LaputinConfiguration(null));
```

Comment:
LaputinConfiguration takes currently the configFilePath. Not sure what would be the way in TypeScript to achieve passing that nicely. Looks like TypeScript supports only 1 constructor per class.